src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/HtmlCSharp/LinkedEditingRangeHandlerTest.cs

```diff
@@ -1,281 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT license. See License.txt in the project root for license information.
-
-using System;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.VisualStudio.LanguageServer.Client;
-using Microsoft.VisualStudio.LanguageServer.ContainedLanguage;
-using Microsoft.VisualStudio.LanguageServer.Protocol;
-using Moq;
-using Xunit;
-using Range = Microsoft.VisualStudio.LanguageServer.Protocol.Range;
-
-namespace Microsoft.VisualStudio.LanguageServerClient.Razor.HtmlCSharp
-{
-    public class LinkedEditingRangeHandlerTest : HandlerTestBase
```

Comment:
I made sure all the deleted test scenarios were covered in the new test class (minus projection/mapping tests which are no longer needed).